anthem/lyrics/loaders.py

```diff
@@ -1,13 +1,68 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016-2017 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
-
 import unicodecsv as csv
 import os
 from past.builtins import basestring
 
 from ..exceptions import AnthemError
 from . import modules
+from .records import switch_company
+
+
+def load_model_csv(ctx, path):
+    """ Use to load any model
+
+    Usage::
+
+        @anthem.log
+        def import_res_partner(ctx):
+            load_model_csv(ctx, 'res.partner.csv')
+
+    """
+    model = os.path.splitext(os.path.basename(path))[0]
+    load_csv(ctx, model, path)
+
+
+def load_users_csv(ctx, path):
+    """ Use to load users without sending emails
+
+    Usage::
+
+        @anthem.log
+        def import_res_users(ctx):
+            load_users_csv(ctx, 'res.users.csv')
+
+    """
+    # make sure we don't send any email
+    model = ctx.env["res.users"].with_context(
+        {"no_reset_password": True, "tracking_disable": True}
+    )
+    load_csv(ctx, model, path)
+
+
+def load_warehouses(ctx, company, path):
+    """ Use to load warehouses in multi-company
+
+    In multicompany mode we must force the company otherwise the sequences that stock
+    module generates automatically will have the wrong company assigned.
+
+    Usage::
+
+        @anthem.log
+        def import_warehouses(ctx):
+            load_warehouses(ctx, 'stock.warehouse.csv')
```

Comment:
The company parameter is missing in "Usage".

Reply:
good catch